### DISCUSSION TITLE
[Projects Beta]  Linked PRs are missing

### DISCUSSION THREAD

**Discussion by amacneil** [community/community#5844] [Projects and Issues] [answered]
In the old Github Issues project board, linked PRs were shown for each issue directly underneath that issue.

In the new issues beta, I can't see the linked PRs for each issue or any way to show them.

Am I missing a setting somewhere or is this just a regression in functionality?

**Comment by katelynnbarlowe** [comment]
We ended up adding a text field to drop in the PR url manually, but this is definitely not ideal. Maybe something that will be improved when the full workflows functionality comes out, but as is, it's not great.

**Comment by arwidt** [comment]
We use this allot, I would also like this feature added to new board layout.

This is BETA:
![Screenshot 2021-10-10 at 16 57 05](https://user-images.githubusercontent.com/253184/136701266-ac60ba9f-4b74-42c2-adcb-0ceda109518a.png)

Old projects:
![Screenshot 2021-10-10 at 16 56 12](https://user-images.githubusercontent.com/253184/136701318-830371a7-d04a-4421-9a02-b98dc7eee86b.png)

I would like all the functionality thats in old project board to be added in the new board view.

**Reply by VMois** [reply]
This is an important feature for my team. One that prevents us from moving to new project boards.

**Comment by wagoodman** [comment]
It would be great to add this feature back as it's very useful when it comes to "status"... though could be a good time to take a look at the UX a bit for it. The PR section has always (to me) seemed a little verbose; there usually isn't a need to see "opened by <author>" and isn't very glanceable as a whole. 

Suggestion: maybe stick with the "pill" pattern on the card like is done with labels an other attributes that only contains the PR icon + a number. When you hover over the pill you'd get a dialog that shows the title, repo, author... similar to what is shown today.

**Comment by Avaq** [comment]
Some duplicates: 

- https://github.com/github/feedback/discussions/5535
- https://github.com/github/feedback/discussions/5003
- https://github.com/github/feedback/discussions/4714

**Comment by azenMatt** [comment, marked as answer]
Hi everyone - we just released this feature! 

You can check it out here - https://github.blog/changelog/2021-12-16-the-new-github-issues-12-16-update/

![changelog_-_linked_prs_v2 2c](https://user-images.githubusercontent.com/7584089/146577210-abb56ee5-37da-45eb-81a6-0b05f92fbb48.gif)

**Reply by VMois** [reply]
Thank you!
